elm-src/NixOptions.elm

```diff
@@ -22,8 +22,8 @@ option_sort_key ( name, option ) =
 
 option_filter : List String -> ( String, Option ) -> Bool
 option_filter terms ( name, option ) =
-    List.any (term_matches name) terms
-        || List.any (term_matches option.description) terms
+    List.all (term_matches name) terms
+        || List.all (term_matches option.description) terms
```

Comment:
Hm, so now (all terms appear in name) OR (all terms appear in description). Is that the intended semantic?

Reply:
Seems like you want:
```
List.all (\term -> term_matches name term || term_matches option.description term) terms
```
This ensures every term is either in the name or description.

Reply:
👍 Got it! Thank you!